<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos_2.qrc">
    <normaloff>:/imagenes/Recursos/salario.png</normaloff>:/imagenes/Recursos/salario.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(39, 145, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(147, 214, 123);</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Nombre:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="inNombre">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(232, 251, 225);</string>
            </property>
            <property name="placeholderText">
             <string>Ingrese el nombre del obrero</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Horas:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="inHoras">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(232, 251, 225);</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(147, 214, 123);</string>
         </property>
         <property name="title">
          <string>Jornada</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="inMatutino">
            <property name="text">
             <string>Matutina</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="inVespertina">
            <property name="text">
             <string>Vespertina</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="inNocturna">
            <property name="text">
             <string>Nocturna</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="cmdCalcular">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(232, 251, 225);</string>
            </property>
            <property name="text">
             <string>&amp;Calcular</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="outResultado">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(147, 214, 123);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(147, 214, 123);</string>
   </property>
   <widget class="QMenu" name="menu_Archivo">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(39, 145, 0);</string>
    </property>
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="action_Nuevo"/>
    <addaction name="actionA_brir"/>
    <addaction name="action_Guardar"/>
    <addaction name="action_Calcular"/>
    <addaction name="action_Salir"/>
   </widget>
   <widget class="QMenu" name="menuAy_uda">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(39, 145, 0);</string>
    </property>
    <property name="title">
     <string>Ay&amp;uda</string>
    </property>
    <addaction name="actionA_cerca_de"/>
   </widget>
   <addaction name="menu_Archivo"/>
   <addaction name="menuAy_uda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Nuevo">
   <property name="icon">
    <iconset resource="recursos_2.qrc">
     <normaloff>:/imagenes/Recursos/agregar-archivo.png</normaloff>:/imagenes/Recursos/agregar-archivo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionA_brir">
   <property name="icon">
    <iconset resource="recursos_2.qrc">
     <normaloff>:/imagenes/Recursos/boton-abrir-archivo.png</normaloff>:/imagenes/Recursos/boton-abrir-archivo.png</iconset>
   </property>
   <property name="text">
    <string>A&amp;brir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Guardar">
   <property name="icon">
    <iconset resource="recursos_2.qrc">
     <normaloff>:/imagenes/Recursos/guardar-el-archivo.png</normaloff>:/imagenes/Recursos/guardar-el-archivo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Calcular">
   <property name="icon">
    <iconset resource="recursos_2.qrc">
     <normaloff>:/imagenes/Recursos/hoja-de-balance.png</normaloff>:/imagenes/Recursos/hoja-de-balance.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Calcular</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_Salir">
   <property name="icon">
    <iconset resource="recursos_2.qrc">
     <normaloff>:/imagenes/Recursos/cerrar-sesion.png</normaloff>:/imagenes/Recursos/cerrar-sesion.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionA_cerca_de">
   <property name="icon">
    <iconset resource="recursos_2.qrc">
     <normaloff>:/imagenes/Recursos/boton-de-informacion.png</normaloff>:/imagenes/Recursos/boton-de-informacion.png</iconset>
   </property>
   <property name="text">
    <string>A&amp;cerca de</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="recursos_2.qrc"/>
 </resources>
 <connections/>
</ui>
